packages/react/src/patterns/sub-patterns/Layout/__stories__/Layout.stories.js

```diff
@@ -86,123 +77,101 @@ storiesOf('Patterns (Sub-Patterns)|Layout', module)
-                      tellus in hac. Rhoncus dolor purus non enim. Suspendisse
-                      sed nisi lacus sed viverra. Faucibus vitae aliquet nec
-                      ullamcorper sit amet risus nullam eget. Augue interdum
-                      velit euismod in pellentesque massa. Fermentum iaculis eu
-                      non diam phasellus vestibulum lorem sed. Volutpat maecenas
-                      volutpat blandit aliquam etiam erat velit scelerisque.
-                    </p>
-
-                    <p style={{ paddingBottom: '1rem' }}>
-                      Mi in nulla posuere sollicitudin aliquam ultrices
-                      sagittis. Elit pellentesque habitant morbi tristique
-                      senectus. Nunc non blandit massa enim nec dui nunc mattis
-                      enim. In egestas erat imperdiet sed euismod nisi porta
-                      lorem mollis. Donec pretium vulputate sapien nec sagittis
-                      aliquam malesuada bibendum arcu. Auctor urna nunc id
-                      cursus metus aliquam eleifend mi. Velit aliquet sagittis
-                      id consectetur. Arcu cursus vitae congue mauris. Gravida
-                      rutrum quisque non tellus orci ac. At erat pellentesque
-                      adipiscing commodo elit at. Amet tellus cras adipiscing
-                      enim eu turpis egestas. Purus gravida quis blandit turpis
-                      cursus in hac habitasse. Interdum velit laoreet id donec
-                      ultrices tincidunt arcu non sodales. Interdum posuere
-                      lorem ipsum dolor sit. Quam quisque id diam vel quam
-                      elementum pulvinar etiam non. Massa enim nec dui nunc
-                      mattis enim. Ac tortor vitae purus faucibus ornare
-                      suspendisse sed. Non pulvinar neque laoreet suspendisse.
-                    </p>
-                  </div>
-                  <div
-                    data-sticky={select(
-                      'Sticky left column',
-                      ['true', 'false'],
-                      'true'
-                    )}
-                    style={{
-                      backgroundColor: 'white',
-                    }}>
-                    <h3>Column 1</h3>
-
-                    <ul className="bx--list--unordered">
-                      <li className="bx--list__item">Item 1</li>
-                      <li className="bx--list__item">Item 2</li>
-                      <li className="bx--list__item">Item 3</li>
-                      <li className="bx--list__item">Item 4</li>
-                      <li className="bx--list__item">Item 5</li>
-                      <li className="bx--list__item">Item 6</li>
-                      <li className="bx--list__item">Item 7</li>
-                    </ul>
-                  </div>
-                </Layout>
-              </div>
+          <Layout type={'2-1'} nested={true}>
```

Comment:
This is where the optional border needs to go.

Reply:
I applied this on the latest version